apps/api/src/app/controllers/command/command-payload-schemas.spec.ts

```diff
@@ -17,16 +13,13 @@ import { Ctor } from '../../../lib/types/Ctor';
 type CommandTypeAndCtor = [string, Ctor<ICommand>];
 
 const commandTypesAndCtors: CommandTypeAndCtor[] = [
-    ['CREATE_SONG', CreateSong],
```

Comment:
This should still be there.

Reply:
Really, we need to discover these dynamically or at least use `createTestModule` for now.